src/tbb/allocator.cpp

```diff
@@ -218,15 +225,16 @@ void __TBB_EXPORTED_FUNC deallocate_memory(void* p) {
 }
 
 bool __TBB_EXPORTED_FUNC is_tbbmalloc_used() {
-    if (allocate_handler == &initialize_allocate_handler) {
-        void* void_ptr = allocate_handler(1);
-        deallocate_handler(void_ptr);
+    auto alloc_handler = allocate_handler.load(std::memory_order_acquire);
```

Comment:
```suggestion
    auto handler_snapshot = allocate_handler.load(std::memory_order_acquire);
```
The `alloc_handler` and `allocate_handler` seem so similar, so I want to propose to improve readability by changing the local copy name.

Reply:
Applied